paper-badge.html

```diff
@@ -28,7 +34,7 @@
 
     <div>
       <paper-button id="btn">Status</paper-button>
-      <paper-badge for="btn" label="♥︎"></paper-badge>
+      <paper-badge icon-badge for="btn" label="favorite"></paper-badge>
```

Comment:
Sooo, I think this should be an `icon='iconName'` kind of attribute. I think it's weird for `label` to change meanings based on the presence of another attribute. Also `label` tends to have an aria meaning (it's going to be read), which in this case it won't